tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+#
+# Smoke test for building, installing and basic usage of tensorboard pip package.
+#
+# Usage:
+#   pip_smoke_test.sh [--python3] [--retries <NUM_RETRIES>] [--port <PORT>]
+
+set -e
+
+die() {
+  echo "$1" 1>&2
+  exit 1
+}
+
+PY_VERSION=2
+TEST_PORT=6006
+NUM_RETRIES=20
+while true; do
+  if [[ -z "$1" ]]; then
+    break
+  fi
+
+  if [[ "$1" == "--python3" ]]; then
+    PY_VERSION=3
+  elif [[ "$1" == "--retries" ]]; then
+    NUM_RETRIES="$2"
+    shift
+  elif [[ "$1" == "--port" ]]; then
+    TEST_PORT="$2"
+    shift
+  else
+    die "ERROR: Unrecognized argument $1"
+  fi
+  shift
+done
+
+echo
+echo "=== Performing Smoke test of tensorboard PIP package ==="
+echo "Settings:"
+echo "  PY_VERSION=${PY_VERSION}"
+echo "  TEST_PORT=${TEST_PORT}"
+echo "  NUM_RETRIES=${NUM_RETRIES}"
+echo
+
+# If virtualenv is not installed, install it.
+if [[ -z $(which virtualenv) ]]; then
+  echo
+  echo "Installing virtualenv"
+  echo
+  pip install virtualenv
+fi
+
+if [[ ${PY_VERSION} == 2 ]]; then
+  pip install wheel
```

Comment:
Also, it installs wheel inside virtualenv to ensure a sandboxed test environment.